### ISSUE TITLE
sitewide 508-defect-1 ❗ Launchblocker [SCREENREADER]: role="alert" MUST not be applied on non-urgent information

### ISSUE THREAD

**Issue by jenstrickland** [department-of-veterans-affairs/va.gov-team#11910]
# [508-defect-1 :exclamation: Launchblocker](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-1)

```diff
! Team affected: VSA Authenticated Experience; Project: Profile 2.0
```

**Feedback framework**
- **❗️ Must** for if the feedback must be applied 
- **⚠️Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Description

When the Account Security screen loads, the first thing the screen reader reads aloud is:

> Have questions about signing in to VA.gov? Get answers to frequently asked questions about how to sign in, common issues with verifying your identity, and your privacy and security on VA.gov.

This is because `role="alert"` was applied to this area.

## Point of Contact

**VFS Point of Contact:** Jennifer

## Acceptance Criteria

As a screen reader user, I want to review the content of the page and alerts only for items of significant relevance.

## Environment

* Operating System: all
* Browser: all
* Screenreading device: all
* Server destination: staging

## Steps to Recreate

1. Start screen reader
1. Logged in, enter https://staging.va.gov/profile/account-security in browser
1. Verify that the Note about Have questions about signing in… is the first thing read

## Possible Fixes (optional)

Remove `role="alert"` 

**Current code**
```diff
- <div class="usa-alert usa-alert-info background-color-only medium-screen:vads-u-margin-top--4" role="alert">
  <div class="usa-alert-body">
    <h3 class="usa-alert-heading">Have questions about signing in to VA.gov?</h3>
    <div class="usa-alert-text">
      <p>Get answers to frequently asked questions about how to sign in, common issues with verifying your identity, and your privacy and security on VA.gov.</p>
      <a href="/sign-in-faq/">Go to VA.gov FAQs</a>
    </div>
  </div>
</div>
```

**Fix**
```diff
+ <div class="usa-alert usa-alert-info background-color-only medium-screen:vads-u-margin-top--4">
  <div class="usa-alert-body">
    <h3 class="usa-alert-heading">Have questions about signing in to VA.gov?</h3>
    <div class="usa-alert-text">
      <p>Get answers to frequently asked questions about how to sign in, common issues with verifying your identity, and your privacy and security on VA.gov.</p>
      <a href="/sign-in-faq/">Go to VA.gov FAQs</a>
    </div>
  </div>
</div>
```

## WCAG or Vendor Guidance (optional)

* [MDN Using the alert role](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_alert_role)

**Comment by erikphansen** [comment]
Also due to the change to AlertBox.

**Comment by mshea0606** [comment]
waiting on platform to make changes, Erik is going to update unit tests

**Comment by erikphansen** [comment]
[This PR reverts the change](https://github.com/department-of-veterans-affairs/veteran-facing-services-tools/pull/363). When that is merged and published we'll need to update `vets-website` to use it.

**Comment by mshea0606** [comment]
This ticket is for the platform team. Our team does not have work to do for this ticket.

**Comment by jenstrickland** [comment]
I adjusted the tags and title, assigned to the frontend team to triage, and assigned @1Copenut since he's on top of getting that other issue rolled back.

**Comment by 1Copenut** [comment]
@jenstrickland This work is underway in the design system package here: https://github.com/department-of-veterans-affairs/veteran-facing-services-tools/pull/363/

**Comment by erikphansen** [comment]
This'll be fixed with [this PR](https://github.com/department-of-veterans-affairs/vets-website/pull/13795)

**Comment by erikphansen** [comment]
@jenstrickland this should be fixed in staging now. Please validate and close at your leisure.

**Comment by jenstrickland** [comment]
Validated! 🎉
